### PR TITLE
finished data preparation and dsr package example

### PR DESCRIPTION
**Pull request by lizacoyer**
- Finished data preparation
- Had two add two example datasets (but let me know if this is a problem, can create this manually in code as well)
- Added dsr example to calculate directly standardized rates, rate ratios, and rate differences (age & sex standardization)

### CODE REVIEW

File: pages/age_standardization.Rmd [hunk 1]
```diff
@@ -30,116 +30,260 @@ knit: (function(inputFile, encoding) {
...
+```{r} eval=FALSE}
+# Another solution that may work
 require(devtools)
-install_version("dsr", version="0.2.2", repos="http:/cran.us.r.project.org")
+devtools::install_version("dsr", version="0.2.2", repos="http:/cran.us.r.project.org")
 ```
 
 For Mac users:
-```{r}
+```{r} eval=FALSE}
```
Comment (nsbatra) [inline pages/age_standardization.Rmd:81]: watch out here - eval = F must be within the brackets

Comment (lizacoyer) [reply pages/age_standardization.Rmd:81]: Ok thanks have changed it to {eval=FALSE}

**Comment by lizacoyer** [comment]
@nsbatra Does it work now?

**Comment by nsbatra** [comment]
@lizacoyer Please go ahead and merge it. We can troubleshoot easier from that state.

**Review by nsbatra — APPROVED** [review]

**Pull request merged**